source/adios2/CMakeLists.txt

```diff
@@ -71,15 +71,16 @@ endif()
   
 if(ADIOS2_HAVE_BZip2)
   find_package(BZip2 REQUIRED)
-  target_sources(adios2 PRIVATE transform/compression/BZip2.cpp)
-  target_link_libraries(adios2 PRIVATE BZip2::BZip2)
+  target_sources(adios2 PRIVATE transform/compress/CompressBZip2.cpp)
+  target_include_directories(adios2 PUBLIC ${BZIP2_INCLUDE_DIR})
```

Comment:
Is this the right way to do it?: 

if(ADIOS2_HAVE_BZip2)
  find_package(BZip2 REQUIRED)
  target_sources(adios2 PRIVATE transform/compress/CompressBZip2.cpp)
  target_link_libraries(adios2 PRIVATE BZip2::BZip2)
endif()